ngx-fudis/projects/ngx-fudis/src/lib/components/typography/body-text/body-text-theme.scss

```diff
@@ -9,6 +10,10 @@ fudis-body-text {
   &:has(+ fudis-body-text) {
     margin-bottom: spacing.$spacing-xs;
   }
+
+  .fudis-link {
```

Comment:
Even though this is quite self-explanatory addition for the styles, I would recommend adding a comment here.
 
These `theme` files can get hard to read over time and the changes are always made for some specific workaround or exception --> comments make it easier to keep track what is still relevant.